backend/handler/handler.go

```diff
@@ -144,6 +149,25 @@ func (h *Handler) Register(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusBadRequest, err)
 	}
 
+	// Validation
+	validate := validator.New()
+	if err := validate.Struct(req); err != nil {
+		fmt.Println(err)
```

Comment:
debugging?

Reply:
I will comment this out. Had it for debugging yesterday.

Reply:
sorry i meant we should push this once this edit is changed